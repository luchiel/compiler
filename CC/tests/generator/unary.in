void main()
{
int a;
a = -5;
printf("%d ", a);
a = -a;
printf("%d ", a);
a++;
printf("%d ", a);
++a;
printf("%d ", ++a);
--a;
printf("%d ", a--);
printf("%d ", a);
}