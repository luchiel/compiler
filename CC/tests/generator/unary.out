-5 5 6 8 7 6 